### PR TITLE
Add tuning option and enable it in nightly testing

### PR DESCRIPTION
**Pull request by jerryyin**
This will address https://github.com/ROCmSoftwarePlatform/llvm-project-private/issues/330

 - Added tuning option in `miopen_validate.sh`, that exports the additional environment variable
 - Added tuning axis in the `Jenkinsfile`

### CODE REVIEW

**Comment by yiqian1** [comment]
Has the MIOpen develop branch already fully supported tuning?

**Comment by jerryyin** [comment]
@yiqian1 It doesn't fully support tuning, but it already support nonxdlops fwd. xdlops fwd is coming in the way. By using develop branch it will automatically pick up tuning coverage as more of my PR get merged. If MIOpen turns out to be merging PRs not as fast as we wish, I will consider of creating dev branch. Bus as of right now it seems that rest of my PRs should get merged latest in 1 or two weeks.

**Review by yiqian1 — APPROVED** [review]

File: mlir/utils/jenkins/miopen-tests/miopen_validate.sh [hunk 1]
```diff
@@ -12,7 +13,7 @@ function usage() {
     cat <<END
 $0: [-d | --direction] DIR [-t | --dtype] [fp16 | fp32]
 [-l | --layout] LAYOUT [-x | --xdlops] [-X | --no-xdlops (default)]
-[--driver DRIVER (default bin/MIOpenDriver)]
+[--no-tuning (default)] [--driver DRIVER (default bin/MIOpenDriver)]
```
Comment (krzysz00) [inline mlir/utils/jenkins/miopen-tests/miopen_validate.sh]: Could we get this into a `[ --tuning  | --no-tuning (default) ]` for the docstring?

**Review by krzysz00 — APPROVED** [review]

File: mlir/utils/jenkins/miopen-tests/miopen_validate.sh (hunk 1 above)
Comment (jerryyin) [reply mlir/utils/jenkins/miopen-tests/miopen_validate.sh]: Sure, done in 4bb1356

**Review by krzysz00 — APPROVED** [review]

**Review by krzysz00 — APPROVED** [review]
Seems like a reasonable refactoring, LGTM

**Pull request merged**